Gemfile.lock

```diff
@@ -644,4 +644,4 @@ RUBY VERSION
    ruby 3.1.0p0
 
 BUNDLED WITH
-   2.1.4
+   2.3.26
```

Comment:
Can you update/add to .tool-version please

Reply:
done